app/models/playbook/config_type.rb

```diff
@@ -0,0 +1,38 @@
+class Playbook::ConfigType < ActiveRecord::Type::Value
+  attr_accessor :backstory, :headings
+  # questions - questions we ask player
+  # options - question options
+
+  def cast(value)
+    return Playbook::ConfigType.new if value.nil?

```

Comment:
Right here. If the value is `nil` (which is what we want if nothing is set), we want this to return `nil`
```suggestion
    return nil if value.nil?

```
Otherwise, it doesn't do the follow-up cast with the value, so we are always getting an empty one.